app/assets/javascripts/application.js.erb

```diff
@@ -319,6 +263,12 @@ class _LobstersFunction {
     fetchWithCSRF("/stories/" + li.getAttribute("data-shortid") + "/" + act, {method: 'post'});
   }
 
+  modalRemoval() {
```

Comment:
Rename to put the verb first and be explicit about the subject: `removeFlagModal`.